### DISCUSSION TITLE
WiFi setting with HTTP server

### DISCUSSION THREAD

**Discussion by Luke1962** [roleoroleo/yi-hack-MStar#445] [Ideas]
I'd like to use my phone as router (hotspot) for the camera. Unfortunately the YI App , during initial configuration, does not allow to select as Hotspot the one of phone itself from which Im configuring the camera.
Is it possible to define somewhere the WiFi credentials? Or even better: to allow changes directly on webcam server page (once hacked)?
p.s. my model is a 9FUSY252IB....   and compliment for the cleanness of User interface !

**Comment by roleoroleo** [comment]
There is a page where you can change wifi credentials. You will find the link in the main page.
